Dockerfile

```
@@ -1,26 +1,22 @@
-FROM hashicorp/terraform:0.14.7
+ARG TERRAFORM_VERSION=0.14.7
+ARG RUBY_VERSION=3.1.2
 
-ENV BUILD_PACKAGES bash ruby-dev build-base
-ENV RUBY_PACKAGES ruby ruby-bundler
+FROM hashicorp/terraform:${TERRAFORM_VERSION} as terraform
 
-# Update and install all of the required packages.
-# At the end, remove the apk cache
-RUN apk --no-cache add $BUILD_PACKAGES && \
-    apk --no-cache add $RUBY_PACKAGES && \
-    gem install bigdecimal:3.0.2 && \
-    rm -rf /var/cache/apk/*
-
-RUN mkdir /usr/action
+FROM ruby:${RUBY_VERSION}
```

I thought this was going to be:
```suggestion
FROM ruby:${RUBY_VERSION}-alpine
```
was there a reason for changing to `bullseye`? Happy to switch to deb, but just want to check if there were any requirements

---

Nope, very happy with Alpine if you all are!

---

Using Debian isn't increasing the image size much, and the ca-certs are available. 
Alongside this, there are some Gems which are only compiled for non-alpine distros